sdk/samples/storage/blobs/src/blobs_client_example.c

```diff
@@ -37,14 +37,25 @@ static az_span content_to_upload = AZ_SPAN_LITERAL_FROM_STR("Some test content")
 #endif
 
 // Enable logging
-static void test_log_func(az_log_classification classification, az_span message)
+static bool test_log_should_write_func(az_log_classification classification)
```

Comment:
In the main PR I had a comment about naming, especially about removing the "test" from the name.